src/main/kotlin/no/nav/familie/ef/sak/økonomi/UtbetalingsoppdragGenerator.kt

```diff
@@ -27,9 +29,10 @@ object UtbetalingsoppdragGenerator {
      * @param[forrigeTilkjentYtelse] Forrige tilkjent ytelse, med fullstendig sett av andeler med id
      * @return Ny tilkjent ytelse med andeler med id'er, samt utbetalingsoppdrag
      */
-    fun lagTilkjentYtelseMedUtbetalingsoppdrag(nyTilkjentYtelse: TilkjentYtelse,
+    fun lagTilkjentYtelseMedUtbetalingsoppdrag(nyTilkjentYtelseMedMetaData: TilkjentYtelseMedMetaData,
```

Comment:
Når man renamer params, variabler, metodenavn er det fint å bruke rename i intellij, min hotkey for det er `fn+shift+f6`, mulig dere har en annen hotkey.
Men, fordelen med det er att den endrer alle referenser, nå ble ikke referensene i docen noen rader opp oppdatert

Reply:
447fa46